src/actions.ml

```diff
@@ -327,22 +306,61 @@ let job_action ~bot_info (job_info : job_info) ~gitlab_mapping =
       failwith "Could not match project name on repository url.\n" ;
     (Str.matched_group 1 repo_url, Str.matched_group 2 repo_url)
   in
-  let repo_full_name = owner ^ "/" ^ repo in
+  let gitlab_repo_full_name = owner ^ "/" ^ repo in
   let gh_owner, gh_repo =
-    github_repo_of_gitlab_project_path ~gitlab_mapping repo_full_name
+    github_repo_of_gitlab_project_path ~gitlab_mapping gitlab_repo_full_name
   in
   let github_repo_full_name = gh_owner ^ "/" ^ gh_repo in
   if String.equal job_info.build_status "failed" then
     let failure_reason = Option.value_exn job_info.failure_reason in
-    job_failure ~bot_info job_info pr_num (gh_owner, gh_repo)
-      github_repo_full_name repo_full_name context failure_reason
+    job_failure ~bot_info job_info ~pr_num (gh_owner, gh_repo)
+      ~github_repo_full_name ~gitlab_repo_full_name ~context ~failure_reason
   else if String.equal job_info.build_status "success" then
     job_success ~bot_info (gh_owner, gh_repo) job_info github_repo_full_name
-      repo_full_name context
+      gitlab_repo_full_name context
     <&> push_status_check ~bot_info (gh_owner, gh_repo) job_info
-          github_repo_full_name repo_full_name
+          ~github_repo_full_name ~gitlab_repo_full_name
   else Lwt.return ()
 
+let minimize_failed_test ~bot_info ~owner ~repo ~base ~head library_name =
+  ignore bot_info ;
+  ignore owner ;
+  ignore repo ;
+  Lwt_io.printf
+    "Minimizing failed library test %s (success on base commit %s, failure on \
+     head commit %s).\n"
+    library_name base head
+
+let minimize_failed_tests ~bot_info ~owner ~repo ~base ~head =
+  GitHub_queries.get_base_and_head_checks ~bot_info ~owner ~repo ~base ~head
+  >>= function
+  | Error err ->
+      Lwt_io.printf
+        "Error while looking for failed library tests to minimize: %s\n" err
+  | Ok (base_checks, head_checks) ->
+      (* TODO: do not run any test if test-suite is updated. *)
```

Comment:
I think the current behavior is correct and this TODO is not needed: if the test-suite is updated and fails, then we already don't try to minimize.  If the test-suite is updated and succeeds, but the CI still fails, then this means there's a test-case in the CI which is not tracked by the test-suite, right?